homeassistant/components/lcn/__init__.py

```diff
@@ -291,12 +284,18 @@ def device_info(self) -> DeviceInfo | None:
             manufacturer="Issendorff",
             via_device=(
                 DOMAIN,
-                generate_unique_id(self.entry_id, self.config[CONF_ADDRESS]),
+                generate_unique_id(
+                    self.config_entry.entry_id, self.config[CONF_ADDRESS]
+                ),
             ),
         )
 
     async def async_added_to_hass(self) -> None:
         """Run when entity about to be added to hass."""
+        # pylint: disable-next=attribute-defined-outside-init
+        self.device_connection = get_device_connection(
```

Comment:
why do we ignore this?

Reply:
Pylint and mypy were a little hard to convince to accept the line. I've solved it a bit different now.

Reply:
... and they are still hard to convince.

Reply:
Have you tried bribing them :P

Reply:
Defining it on class level seems to work. Thanks for the hint!